### DISCUSSION TITLE
Querying Thruk API "services" for all services that are part of a service group

### DISCUSSION THREAD

**Discussion by mdoakes42** [sni/thruk#1360] [Q&A]
I would like to query the Thruk API for all services related to a specific service group.  I'm able to query on other fields but cannot query on the "groups" field.  It appears it is a "list" field so I think that is the issue.  Does anyone know how to query a "list" field.   My example query is this.   (masking my hostnamne with xxxxx.com) 

I am querying the /services endpoint 

http://xxxxx.com:3000/api/datasources/proxy/uid/fdke20tq8nvnkc/r/v1/services?limit=&q=groups="test_group"  
I just get back []

If I run this query on a different "string" type field I get data as expected.     
http://xxxxx.com:3000/api/datasources/proxy/uid/fdke20tq8nvnkc/r/v1/services?limit=&q=display_name="test service"

If anyone has an example of querying a list field please let me know.  THank you!

**Comment by sni** [comment]
the "trick" on list columns is to use the `>=` operator as mentioned on https://thruk.org/documentation/rest.html#_filtering
Because the `=` is a special character in urls, it's easier to use the `gte` alias, ex.:
Try http://xxxxx.com:3000/api/datasources/proxy/uid/fdke20tq8nvnkc/r/v1/services?limit=&q=groups[gte]=test_group"
